_shared/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+*.dmg
```

Comment:
I think `.DS_Store` is already in [/.gitignore](https://github.com/mozilla-services/services-test/blob/master/.gitignore).

Not sure if we want to add `*.dmg` to the root .gitignore and murder this file completely. **#onegitignoretorulethemall**